docs/user-guide/federation/federation-namespace.md

```diff
@@ -0,0 +1,90 @@
+---
+---
+
+This guide explains how to use events in Federation control plane to help in debugging
+
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+This guide assumes that you have a running Kubernetes Cluster
+Federation installation. If not, then head over to the
+[federation admin guide](/docs/admin/federation/) to learn how to
+bring up a cluster federation (or have your cluster administrator do
+this for you). Other tutorials, for example
+[this one](https://github.com/kelseyhightower/kubernetes-cluster-federation)
+by Kelsey Hightower, are also available to help you.
+
+You are also expected to have a basic
+[working knowledge of Kubernetes](/docs/getting-started-guides/) in
+general and [Namespaces](/docs/user-guide/namespaces/) in particular.
+
+## Overview
+
+Namespaces in federation control plane (refered to as "federation namespaces" in
```

Comment:
Fixed "federation" -> "federated" everywhere in this doc. Also renamed this doc from federation-namespace.md to namespaces.md since this is already in the federation dir